pkg/traceability/agent_test.go

```diff
@@ -98,7 +98,11 @@ func (m mockAnalyticsClient) GetClientApplication(string) (*anypoint.Application
 
 func (m mockAnalyticsClient) OnConfigChange(_ *config.MulesoftConfig) {
```

Comment:
Let's get a test to confirm that the time stamps are saved when the data is fetched, and that the timestamps change on subsequent calls of the function.